api/tests/devcomp/unit/infrastructure/datasources/learning-content/validation/module.js

```diff
@@ -19,6 +19,13 @@ const moduleSchema = Joi.object({
     .regex(/^[a-z0-9-]+$/)
     .required(),
   title: Joi.string().required(),
+  details: Joi.object({
+    image: Joi.string().uri().required(),
+    description: Joi.string().required(),
+    duration: Joi.string().required(),
```

Comment:
Est-ce qu'on ne prévoierait pas un format un peu plus intelligent ?

Par exemple : https://joi.dev/api/?v=17.9.1#stringisoduration

Côté `trainings` on stocke dans PG un format `interval`.

Reply:
Je propose de le stocker en nombre de minutes.

```suggestion
    duration: Joi.number().integer().min(0).max(120).required(),
```

Ça nous permet :
- de traduire correctement
- d'ouvrir la porte au tri/filtre par durée sans que ce soit trop coûteux

Reply:
fait

Reply:
@mirawlin si tu n'avais pas vu